versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx

```diff
@@ -51,14 +58,32 @@ We see in the graph above that there are two usage metrics that LangSmith charge
 The first metric tracks all traces that you send to LangSmith. The second tracks all traces that also have our Extended 400 Day Data Retention.
 For more details, see our [data retention conceptual docs](../../concepts/usage_and_billing/data_retention_billing).
 
+LangSmith Traces usage is measured per workspace. This is because workspaces often represent development environments (as in our example),
+or teams within an organization, and as a LangSmith administrator, we want to know which environment or team is responsible for
+the majority of our costs.
```

Comment:
Maybe break this up as:
`measured per workspace, because workspaces...`
then
`or teams within an organization. As a Langsmith administrator`

'This is because' sound too colloquial